### PR TITLE
[Security] Fix Reflected XSS vulnerability on all NDB_Pages 

### PR DESCRIPTION
**Pull request by johnsaigle**
## This PR sanitizes data in LorisForm, boosting security and removing the security burden from module developers

Previously, it was trivial to inject Javascript into nearly every page as malicious data could be injected via GET requests and served without being sanitized. This makes users vulnerable to CSRF, XSS, and phishing attacks among others.

By sanitizing values in LorisForm, we prevent many of these vulnerabilities. 

I suggest that this fix in combination with #2919, #2826 warrant another release on 17.0.

### CODE REVIEW

**Comment by johnsaigle** [comment]
@llevitis Could you review and confirm this fixes the bug you found in `issue_tracker` a few weeks back?

**Comment by llevitis** [comment]
@johnsaigle the exploits I used previously to catch the XSS vulnerability no longer work! :)

**Comment by johnsaigle** [comment]
@llevitis Awesome! Could you add an approved review?

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-17.0.4
+17.0.5
```
Comment (driusan) [inline VERSION]: This shouldn't have the VERSION file change.

**Comment by johnsaigle** [comment]
@driusan I removed the VERSION file. Please rereview.

**Pull request merged**